src/components/navigation/desktopnavigation/DesktopNavigation.tsx

```diff
@@ -5,72 +5,92 @@ import RoutingPath from '../../../routes/RoutingPath'
 import { useContext } from 'react'
 import { UserContext } from '../../../shared/providers/UserProvider'
 import { Profile } from '../profile/Profile'
+import { DESKTOP_NAV_HEIGHT } from '../../../shared/styles/constants'
```

Comment:
why this external constant? :octocat:

Reply:
@ArastoSahbaei To make room for the navbar so that there's no content hiding underneath it later.
Instead of having multiple references to the height we might as well store it in a variable :smiley_cat: